packages/goto/scripts/postinstall.js

```diff
@@ -9,17 +9,15 @@ const writeFile = promisify(fs.writeFile)
 
 const OUTPUT_FILENAME = 'src/engine.bin'
 
-// Small helpers to allow fetching different types of data with `got`
-const fetchBuffer = async url => (await got(url, { encoding: null })).body
-const fetchText = async url => Buffer.from(await fetchBuffer(url), 'ascii')
-const fetchJson = async url => JSON.parse(await fetchText(url))
-
 // Lightweight `fetch` polyfill on top of `got` to allow consumption by adblocker
 const fetch = url =>
   Promise.resolve({
-    text: () => fetchText(url),
-    arrayBuffer: () => fetchBuffer(url),
-    json: () => fetchJson(url)
+    text: () =>
+      got(url)
+        .text()
+        .then(str => Buffer.from(str, 'ascii')),
```

Comment:
Why is this needed? Doesn't `got(url).text()` already resolve to a `string`?

Reply:
I maintained it since you added it in the past 😄so I guess now I can remove it!

Reply:
I recently discovered Node.js buffer on the browser side is ArrayBuffer, dafuq

Reply:
> I recently discovered Node.js buffer on the browser side is ArrayBuffer, dafuq

It's more `Uint8Array` which wraps around an instance of `ArrayBuffer`. I guess Node.js' `Buffer` is legacy, but at least an instance of `Buffer` is also an instance of `Uint8Array`...

Reply:
> I maintained it since you added it in the past smileso I guess now I can remove it!

I think it was needed before the update because we could not specify the expected return type from `got`! Should not be needed now.

Reply:
👌